features/partners.feature

```diff
@@ -8,7 +8,8 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
   # Dit betekent dat in 'partners' zowel actuele partners zitten als ontbonden partners
   # Een proxy vertaalt dit naar de vorm Persoon, waarin sommige gegevens in bewerkte vorm worden opgenomen en waarin informatievragen kunnen zitten
   # Een voorbeeld van een informatievraag is naam.aanschrijfwijze, waarin op basis van de aanduiding naamgebruik, de naam van de persoon en de naam van de (ex)partner een aanschrijfnaam wordt samengesteld
-  # De proxy verwijdert de partners met datum ontbinding die geleverd zijn in de GbaPersoon
+  # Als er een actuele partner is (huwelijk / partnerschap is niet ontbonden) verwijdert de proxy de partners met datum ontbinding die geleverd zijn in de GbaPersoon
```

Comment:
Een actuele partner heeft geen ontbinding property. Daardoor is deze regel volgens mij niet nodig

Reply:
Wat ik hier bedoel is dat alle ontbonden partners worden verwijderd als er sprake is van een actuele partner. Moet ik dat anders formuleren ?